Quarg content

Brick63 opened the pull request:
Seperate from the base PR

Brick63 committed:
add skylance

add sprite

apply the sprite

apply the hardpoint

add a description

Brick63 committed:
Update copyright

indentation fix

add blade to copyright

Brick63 committed:
add offset

description change

Brick63 committed:
replace hardpoint

slight fix

Brick63 committed:
add anti missile sprite

add anti missile hardpoint

apply sprites

Brick63 committed:
add shield and battery sprites

apply sprites

Brick63 committed:
add small - large engine sprites

it didn't make sense to only make a medium so i made small and large ones aswell

apply medium sprites

Brick63 committed:
add the antimatter core and improve the battery's

sprites

update the antimatter core to use sprite

and be unplunderable

Brick63 committed:
add the countermeasure sprite

apply sprite

Update copyright

Brick63 committed:
minor fix

Brick63 committed:
fix

delete wrongly named sprite

upload renamed sprite

renamed skylance sprite

remove old sprite

rename hardpoint

remove old

apply new sprite

Brick63 committed:
Update copyright

add engines to copyright

Update copyright

Update copyright

Brick63 committed:
rename some sprites

Brick63 committed:
fix

separate turrets

Brick63 committed:
better antimatter core sprite

now with a glow

Brick63 committed:
re add the hull repair

not sure why that went away

Brick63 committed:
shrink hardpoints a bit

hardpoints were too big compared to other turrets and have been shrunk slightly

Brick63 committed:
Merge branch 'master' into skylance

Zitchas reviewed (approved):
Looks good to me. Very nice images.

Amazinite committed:
Normalized spelling of gases

KiLEdEnNis commented:
I'd say redo the engines because they look ugly and you can do a lot better
the rest is fine though

Pointedstick commented:
I agree that the engines are a bit sub-par, but everything else is fantastic.

Nescio0 committed:
update 'hull disabled at' tooltips

Lorantine committed:
typo

elgeonmb committed:
Adds extra response to the "Alondo is gay" revelation (#3705)

* lets you come out to freya winters

* changes freya's wording to be more freya

* Changed the dialog choice to flow better with the conversation

czartrak committed:
Center Pointedstick's Vanguard

Ref: #3954

* Delete pointedstick_vanguard.png

* Add files via upload

* Update persons.txt

warp-core committed:
Typo fix

tehhowch committed:
Add configuration files for CI tools

https://github.com/endless-sky/endless-sky/pull/2786

* Add configuration files for CI tools
 - .travis.yml for build verification of linux executables
 - .appveyor.yml for build verification of Windows executables
    + .winmake for "headless" builds of Endless Sky using only MinGW

* Update .gitignore to exclude more local-only files
 - Adding the CI files so local modifications can be made without committing them back to upstream

* Distribute an .editorconfig file to help maintain repo style
 - Some IDEs / editors have native support, others (Notepad++, Atom, VS Code, Codeblocks) have plugins available

Ferociousfeind committed:
Add hostile militia retorts

#3710

czartrak committed:
Assassination pirate jobs

#3984

TheUnfetteredOne committed:
New spaceport missions

#3151

Amazinite committed:
Blockade running missions

#3298

Amazinite committed:
Hai spaceport missions

#3781

Amazinite committed:
Terraforming Tundra

#3216

Amazinite committed:
Typo fix (@Proont)

Pointedstick committed:
Add a high-tech image for Lagrange

#3897

Fzzr committed:
Add a pirate mission to capture a rogue smuggler

#3475

Vilhelm16 committed:
Added unique explosions for Pug ships

#3957

Brick63 committed:
Added a Remnant H2H weapon

#3960

Brick63 committed:
Added three new Hai ships

#3314

Pointedstick committed:
Tweak default ship and variant loadouts to reflect missile changes

#3922

Amazinite committed:
Error fixes

Pointedstick committed:
Altered the shield/hull balance of the Corvette and Firebird

#3411

tiennou committed:
Fix XCode libraries

https://github.com/endless-sky/endless-sky/pull/3808

* Fix path to brew's SDL2.dylib install 

* Remove unused framework path

* Don't add some Xcode directory to the header search path

"XCode/mumble/jumble/usr/include" should be equivalent to /usr/include
(and the build system takes care of prepending the SDKROOT).

Having this first makes the OS-provided non-jpeg-turbo library to be
picked up.

Brick63 committed:
Merge branch 'master' into skylance

Brick63 committed:
New Engine sprites

Out with the tubes in with the trianglular prisims
